rock-n-rollup/src/core/runtime.rs

```diff
@@ -104,132 +49,127 @@ pub trait Runtime: 'static {
 }
 
 #[derive(Default)]
-pub struct KernelRuntime {}
+pub struct KernelRuntime<R>
+where
+    R: tezos_smart_rollup_host::runtime::Runtime + 'static,
```

Comment:
does it really need to be static?

Reply:
without the `static` it raise an error of `R` not live long enough. As this example

```
error[E0310]: the parameter type `R` may not live long enough
   --> rock-n-rollup/src/core/service.rs:405:13
    |
405 |             service.register(custom_transition);
    |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ ...so that the type `R` will meet its required lifetime bounds
    |
help: consider adding an explicit lifetime bound...
    |
401 |         R: Runtime + 'static,
    |                    +++++++++
```